lib/src/storage_impl.dart

```diff
@@ -181,7 +181,7 @@ class _BucketImpl implements Bucket {
     return sink;
   }
 
-  Future writeBytes(String objectName, List<int> bytes,
+  Future<ObjectInfo> writeBytes(String objectName, List<int> bytes,
```

Comment:
This is a public API and we don't want to expose the `ObjectInfo` as return type. The caller should get nothing! (we should keep the same return value from the implemented interface!)

Reply:
We *do* expose ObjectInfo in the public API...

https://github.com/dart-lang/gcloud/blob/e260cc330f031331ce330a12c2dcedde6eaef5c2/lib/storage.dart#L747